### ISSUE TITLE
[Android] Service not working

### ISSUE THREAD

**Issue by rotorgames** [dbankier/TiShadow#418]
Not working Android service (AlarmManager).

Tried to do so, did not work:

###### /Users/admin/Library/Application Support/Titanium/mobilesdk/osx/3.5.0.GA/android/templates/build/JSIntervalService.java
```java
package <%- appid %>;

import ti.modules.titanium.android.TiJSIntervalService;

public final class <%- service.classname %> extends TiJSIntervalService {
	public <%- service.classname %>() {
		super("appdata-private://my_app/<%- service.url %>");
	}
}
```
###### tiapp.xml
```xml
<services>
     <service type="interval" url="services/boot.js"/>
</services>
```
###### console
```
[INFO]  I/bencoding.AlarmManager: (KrollRuntimeThread) [248,6609] Alarm Service Request Created
[INFO]  I/bencoding.AlarmManager: (main) [2047,8656] Alarm Service Started
[DEBUG] TiJSService: (main) [4,8660] onStartCommand
[WARN]  TiJSIntervalService: (main) [4,8664] The intent is missing the extra value 'interval', therefore the code will be executed only once.
[DEBUG] TiJSService: (main) [1,8665] Eval JS Service:appdata-private://FindMe/services/boot.js
[ERROR] TiAssetHelper: Error while reading asset "appdata-private://FindMe/services/boot.js":
[ERROR] TiAssetHelper: java.io.FileNotFoundException: appdata-private://FindMe/services/boot.js
[ERROR] TiAssetHelper:  at android.content.res.AssetManager.openAsset(Native Method)
[ERROR] TiAssetHelper:  at android.content.res.AssetManager.open(AssetManager.java:316)
[ERROR] TiAssetHelper:  at android.content.res.AssetManager.open(AssetManager.java:290)
[ERROR] TiAssetHelper:  at org.appcelerator.kroll.util.KrollAssetHelper.readAsset(KrollAssetHelper.java:77)
[ERROR] TiAssetHelper:  at ti.modules.titanium.android.TiJSService.executeServiceCode(TiJSService.java:74)
[ERROR] TiAssetHelper:  at ti.modules.titanium.android.TiJSIntervalService.executeServiceCode(TiJSIntervalService.java:47)
[ERROR] TiAssetHelper:  at ti.modules.titanium.android.TiJSService.start(TiJSService.java:97)
[ERROR] TiAssetHelper:  at ti.modules.titanium.android.TiJSService.onStartCommand(TiJSService.java:47)
[ERROR] TiAssetHelper:  at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:2860)
[ERROR] TiAssetHelper:  at android.app.ActivityThread.access$2200(ActivityThread.java:161)
[ERROR] TiAssetHelper:  at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1362)
[ERROR] TiAssetHelper:  at android.os.Handler.dispatchMessage(Handler.java:102)
[ERROR] TiAssetHelper:  at android.os.Looper.loop(Looper.java:157)
[ERROR] TiAssetHelper:  at android.app.ActivityThread.main(ActivityThread.java:5356)
[ERROR] TiAssetHelper:  at java.lang.reflect.Method.invokeNative(Native Method)
[ERROR] TiAssetHelper:  at java.lang.reflect.Method.invoke(Method.java:515)
[ERROR] TiAssetHelper:  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1265)
[ERROR] TiAssetHelper:  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1081)
[ERROR] TiAssetHelper:  at dalvik.system.NativeStart.main(Native Method)
[ERROR] TiAssetHelper: Error while reading asset "appdata-private://FindMe/services/boot.js":
[ERROR] TiAssetHelper: java.io.FileNotFoundException: appdata-private://FindMe/services/boot.js
[ERROR] TiAssetHelper:  at android.content.res.AssetManager.openAsset(Native Method)
[ERROR] TiAssetHelper:  at android.content.res.AssetManager.open(AssetManager.java:316)
[ERROR] TiAssetHelper:  at android.content.res.AssetManager.open(AssetManager.java:290)
[ERROR] TiAssetHelper:  at org.appcelerator.kroll.util.KrollAssetHelper.readAsset(KrollAssetHelper.java:77)
[ERROR] TiAssetHelper:  at org.appcelerator.kroll.runtime.v8.V8Runtime.nativeRunModule(Native Method)
[ERROR] TiAssetHelper:  at org.appcelerator.kroll.runtime.v8.V8Runtime.doRunModule(V8Runtime.java:177)
[ERROR] TiAssetHelper:  at org.appcelerator.kroll.KrollRuntime.handleMessage(KrollRuntime.java:299)
[ERROR] TiAssetHelper:  at org.appcelerator.kroll.runtime.v8.V8Runtime.handleMessage(V8Runtime.java:203)
[ERROR] TiAssetHelper:  at android.os.Handler.dispatchMessage(Handler.java:98)
[ERROR] TiAssetHelper:  at android.os.Looper.loop(Looper.java:157)
[ERROR] TiAssetHelper:  at org.appcelerator.kroll.KrollRuntime$KrollRuntimeThread.run(KrollRuntime.java:112) 
```
Why android can not find `appdata-private: // FindMe / services / boot.js`, but that file is there !?

**Comment by dbankier** [comment]
You need to dig into `TiJSIntervalService.java`. At a quick glance it doesn't support `appdata-private` urls. 

Closing this only as it is a duplicate of #260.
